url/url.cpp

```diff
@@ -1189,8 +1156,6 @@ void UrlParser::state_fragment() {
 
 // https://url.spec.whatwg.org/#concept-domain-to-ascii
 std::optional<std::string> UrlParser::domain_to_ascii(std::string_view domain, bool be_strict) const {
-    icu_init();
```

Comment:
Now that `icu_init` is gone, do we still need `icu_cleanup`?

Reply:
yeah, it's still gonna leak memory and make the tests complain if we don't do that.